bin/alluxio-mount.sh

```diff
@@ -156,10 +169,10 @@ function main {
     Mount|SudoMount)
       case "${2}" in
         ""|local)
-          mount_ramfs_local $1
+          mount_ramfs_local ${1}
```

Comment:
FYI, the Google shell style guide only requires the brace convention for multi-character variable names